vnext/Microsoft.ReactNative/Fabric/Composition/CompositionContextHelper.cpp

```diff
@@ -230,6 +230,13 @@ struct CompVisual : public winrt::implements<
     m_visual.RelativeOffsetAdjustment(relativeAdjustment);
   }
 
+  void RelativeSizeWithOffset(
+      winrt::Windows::Foundation::Numerics::float2 size,
```

Comment:
>size [](http://example.com/codeflow?start=51&length=4)

Since foat2 is a struct, I wonder if it would be more efficient to pass them as const& as you do above in line 218 for the Size().